src/checkout/checkout-service.ts

```diff
@@ -438,6 +439,25 @@ export default class CheckoutService {
         return this._dispatch(action, { queueId: 'shippingCountries' });
     }
 
+    /**
+     * This is still in beta.
```

Comment:
Not sure if we need to mention this here as part of checkout service?

Reply:
I'd say not? 😆